src/main/java/nl/tudelft/contextproject/model/level/GeneratorRoom.java

```diff
@@ -19,11 +19,11 @@
      * Create a Generator Room.
```

Comment:
Indentation is suddenly spaces instead of tabs like the other files. Some kind of IDE mess up?